<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>receiptwindow</class>
 <widget class="QDialog" name="receiptwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {	
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(51, 51, 51, 255), stop:				0.525822 rgba(95, 95, 95, 255), stop:1 rgba(58, 58, 58, 255));
}

QLabel#backLabel {
	background-color: rgb(75, 75, 75);
	border: 6px groove #4c4c4c;
	border-radius: 20px;
}

QLabel#label {
	background: transparent;
	border: none;
	font: 290 16pt &quot;Microsoft YaHei Light&quot;;
	color: #dadada;
	qproperty-geometry:rect(80 40 551 26);
}
QLabel#labelError{
	font: 290 20pt &quot;Microsoft YaHei Light&quot;;
	color: #D10000;
	

}

QLabel#labelSaldo
{
	background: transparent;
	border: none;
	font: 290 18pt &quot;Microsoft YaHei Light&quot;;
	color: #dadada;

}
QLabel#labelSumma
{
	background: transparent;
	border: none;
	font: 290 18pt &quot;Microsoft YaHei Light&quot;;
	color: #dadada;
}
QLabel#label_3
{
	font: 290 16pt &quot;Microsoft YaHei Light&quot;;	
	color: #dadada;
	background-color: qlineargradient(spread:pad, x1:0, y1:0.431818, x2:1, y2:0.409364, stop:0.217877 			rgba(43, 	43, 43, 255), stop:0.50838 rgba(64, 64, 64, 255), stop:0.938547 rgba(37, 37, 37, 255));
	border: 2px groove #4a8390;
	border-radius: 10px;
	padding: 6px;
}
QLabel#label_4
{
	font: 290 16pt &quot;Microsoft YaHei Light&quot;;	
	color: #dadada;
	background-color: qlineargradient(spread:pad, x1:0, y1:0.431818, x2:1, y2:0.409364, stop:0.217877 			rgba(43, 	43, 43, 255), stop:0.50838 rgba(64, 64, 64, 255), stop:0.938547 rgba(37, 37, 37, 255));
	border: 2px groove #4a8390;
	border-radius: 10px;
	padding: 6px;
}

QPushButton {
background: transparent;
border: none;

}
QLabel#pirkkal {	
	color: #dadada;
	font: 350 10pt &quot;Leelawadee UI Semilight&quot;;
	qproperty-geometry:rect(610 550 101 35);
}</string>
  </property>
  <widget class="QLabel" name="pirkkal">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>550</y>
     <width>101</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Pirkka Pankki</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>391</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Nostettu summa:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>391</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Tilisi saldo:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelSumma">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>200</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelSaldo">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>300</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="backLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>611</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>551</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Kuitti</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="eurob1">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>200</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="gifsANDicons.qrc">
     <normaloff>:/icons/icons/euro.png</normaloff>:/icons/icons/euro.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="eurob2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>300</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="gifsANDicons.qrc">
     <normaloff>:/icons/icons/euro.png</normaloff>:/icons/icons/euro.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="labelError">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>backLabel</zorder>
  <zorder>label_4</zorder>
  <zorder>label_3</zorder>
  <zorder>pirkkal</zorder>
  <zorder>labelSumma</zorder>
  <zorder>labelSaldo</zorder>
  <zorder>eurob1</zorder>
  <zorder>eurob2</zorder>
  <zorder>labelError</zorder>
 </widget>
 <resources>
  <include location="gifsANDicons.qrc"/>
 </resources>
 <connections/>
</ui>
